src/blocks/modal.css

```diff
@@ -128,12 +148,6 @@
   opacity: 0.8;
 }
 
-.modal__submit-btn-disabled {
-  background-color: rgba(33, 33, 33, 1);
-  opacity: 0.5;
-  color: rgba(252, 245, 229, 1);
```

Comment:
* If the form is `invalid`, the submit button should have `opacity: 0.5` (to make it look disabled) https://snipboard.io/f4XZ06.jpg 


You shouldn't have deleted these styles of the disabled button. They are needed